inference/core/workflows/core_steps/transformations/image_slicer/v1.py

```diff
@@ -49,7 +49,7 @@ class BlockManifest(WorkflowBlockManifest):
         json_schema_extra={
             "name": "Image Slicer",
             "version": "v1",
-            "short_description": "Splits input image into series of smaller images to perform accurate prediction.",
+            "short_description": "Consolidate detections from multiple sections of an input image into a single detection.",
```

Comment:
wrong description, should be:

`Tile the input image into a list of smaller images to perform small object detection.`